cdba-server.c

```diff
@@ -79,12 +79,14 @@ static struct fastboot_ops fastboot_ops = {
 
```

Comment:
somebody's gonna pull out half of their hair over a nullptr here.. :P

Reply:
?

Reply:
fastboot_ funcs don't do nullchecking

Reply:
Ah... true. I'll fix that.

Reply:
Done